karma-ng.conf.js

```diff
@@ -153,6 +153,7 @@ function makeConfig(packageName, argv) {
       public: 'team',
       startConnect: true,
       connectOptions: {
+        logfile: './sauce.log',
```

Comment:
It seems like the [sauce-connect-launcher](https://github.com/bermi/sauce-connect-launcher) doesn't do a `mkdir -p` if `reports/` doesn't exist so it just causes testing to hang forever ^This was like a two day issue for me -_-